controllers/managedocs_controller.go

```diff
@@ -1278,26 +1280,27 @@ func (r *ManagedOCSReconciler) findOCSVolumeClaims() (bool, error) {
 	if err := r.UnrestrictedClient.List(r.ctx, &storageClassList); err != nil {
 		return false, fmt.Errorf("unable to list storage classes: %v", err)
 	}
-	// create slice of ocs storage classes by checking provisioner
+
+	// create map of ocs storage classes by checking provisioner
 	// Provisioner will be prefixed by namespace
```

Comment:
Let try to capture the intent with the comment, my suggestion is something similar to the following text:

"Create a set of storage classed names who are using the ODF provisioner (provisioner name prefixed with the namespace name)"